PythonScript/src/Utility.cpp

```diff
@@ -0,0 +1,33 @@
+#include "stdafx.h"
+#include "Utility.h"
+
+HBITMAP ConvertIconToBitmap(LPCWSTR file_path) {
+	HBITMAP hBitmap = nullptr;
+	HICON hIcon = (HICON)LoadImage(nullptr, file_path, IMAGE_ICON, 16, 16, LR_COLOR | LR_LOADFROMFILE);

```

Comment:
Maybe DPIManagerV2::loadIcon() could be used here for High DPI support.

Reply:
The size is currently set to 16x16. To support really high dpi, further changes are probably needed, e.g. how to handle multiple bmp sizes.